packages/core/useFps/index.ts

```diff
@@ -12,6 +13,8 @@ export interface UseFpsOptions {
 
 export function useFps(options?: UseFpsOptions): Ref<number> {
   const fps = ref(0)
+  if (!isClient)
+    return fps
   const every = options?.every ?? 10
 
   let last = performance.now()
```

Comment:
Can we just check if `performance` exists?

Reply:
<img width="765" alt="image" src="https://user-images.githubusercontent.com/22515951/153698136-2d6f6891-ae83-4cee-aafb-059c0dce0777.png">

It'll throw ReferenceError , because it should be `Window.performance`

Reply:
Yeah, I mean, if a performance object exists in Node, we shouldn't skip it. We need to do the check `if (typeof performance === 'undefined')` instead of `isClient`

Reply:
Fixed